### DISCUSSION TITLE
Welcome to daq Discussions!

### DISCUSSION THREAD

**Discussion by hewanbing** [attpc-daq/daq#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by susu5767** [comment]
SiTCP软件官方使用说明书下载地址： https://www.bbtech.co.jp/download-files/sitcp/index.html；
根据此使用说明书可以熟悉一些数据获取流程。
[SiTCPUtilityUserGuide.0.7.3.pdf](https://github.com/attpc-daq/daq/files/9167330/SiTCPUtilityUserGuide.0.7.3.pdf)
